src/mappings/DAOToken/schema.graphql

```diff
@@ -11,12 +11,12 @@ type TokenHolder @entity {
 	contract: Bytes!
 	address: Bytes!
 	balance: BigInt!
-	allowances: [Allowance!] @derivedFrom(field: "owner")
 }
 
 type Allowance @entity {
 	id: ID!
-	owner: TokenHolder!
+	contract: Bytes!
```

Comment:
I guess 'contract' is the 'token' address . right ?
how about rename that to 'token' ?
@ben-kaufman